src/reporters/xhr.coffee

```diff
@@ -12,6 +12,7 @@ report = (notice, opts) ->
   req.onreadystatechange = ->
     if req.readyState == 4 and req.status == 201 and console?.debug?
       resp = JSON.parse(req.responseText)
+      promise.resolve({id: resp.id})
       console.debug("airbrake: error #%s was reported: %s", resp.id, resp.url)
```

Comment:
Wholly unrelated but I noticed recently the IE9 does not support `console.debug`. Do we have a polyfill in place perhaps?